src/mr-importer/loader.cpp

```
@@ -1566,14 +1570,200 @@ static Model::Lights get_lights_from_asset(fastgltf::Asset *asset)
+  }
+  if (try_wuffs_file(path_str, new_image))
+    goto format_fixup;
+  if (load_dds_from_file_path(path_str, new_image))
+    goto format_fixup;
+  if (load_ktx2_from_file_path(path_str, new_image))
+    goto format_fixup;
+
+  return std::nullopt;
+
+format_fixup:
+  if (new_image.format == vk::Format()) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)